app/main.py

```diff
@@ -1,8 +1,32 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.add_person_to_the_list(self)
+
+    @classmethod
+    def add_person_to_the_list(cls, person: object) -> None:
+        cls.people[person.name] = person
 
```

Comment:
BTW, it can be simlified to Person.people[name] = self, but your solution is good as well.